pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[project]
+name = "lsst-ctrl-execute"
+requires-python = ">=3.11.0,<3.13.0"
```

Comment:
I've been testing all middleware with 3.13 so I would only constrain it to 3.13 if you really think it's completely broken with 3.13. (and add 3.13 to the github action).

Reply:
`htcondor` is not available for 3.13, so I had to back it off.

Reply:
I was meaning to ask why ctrl_execute depended on lsst-bps-htcondor. What's the reasoning for that? ctrl_execute doesn't use any middleware itself.

Reply:
I had not realized that this package depended on a utility function from ctrl_bps_htcondor that isn't part of the normal BPS API.